authserver/api/static/css/style.css

```diff
@@ -1,51 +1,64 @@
-html,
-body {
+:root {
+  --blue: #0fbaa8;
+  --light-blue: #0fbaa8b3;
+  --gray20: #333333;
+}
+
+html {
   height: 100%;
+  font-size: 100%;
```

Comment:
For your comment about font-size being 18px in the docs, i learned something just now that can help adjust font-sizes more easily with `rem`. by setting `font-size: 62.5%`, this makes calculating `rem` easier because it sets the base font-size to 10px, so if the mock ups have 18px fonts then you can just set `1.8rem`, much easier than calculating `18/16 rem`

Reply:
you'd have to update all the other `rem`s though